tools/nl/embeddings/input/base/sheets_svs.csv

```diff
@@ -3605,4 +3605,12 @@ worldBank/EG_ELC_ACCS_ZS,percentage of population with access to electricity
 sdg/SG_GEN_PARL.SEX--F,proportion of seats held by women in national parliaments
 Count_Person_Upto18Years,children population count
 Count_Person_18OrMoreYears,adult population count
-Count_Person_65OrMoreYears,senior population count
\ No newline at end of file
+Count_Person_65OrMoreYears,senior population count
+Count_VehicleCrashIncident_Dark,number of vehicle accident in the dark
+Count_VehicleCrashIncident_Daylight,number of vehicle accident in the daylight
+Count_VehicleCrashIncident_RainOrMist,number of vehicle accident in rain or mist
+Count_VehicleCrashIncident_SleetOrHail,number of vehicle accident in sleet or hail
+Count_VehicleCrashIncident_Snow,number of vehicle accident in snow
+Count_VehicleCrashIncident_InvolvedInCrash_SchoolBus,number of school bus accident

```

Comment:
to be consistent with the rest of the descriptions, maybe something like "number of vehicle accident with school bus" or "number of vehicle accident involving school bus"

Reply:
Modified to "number of vehicle accident involving school bus"